apps/server/src/controllers/integrationController.ts

```diff
@@ -174,6 +174,19 @@ export function dispatchFromAdapter(
       PlaybackService.roll();
       break;
     }
+    case 'addtime': {
```

Comment:
I wonder if `addtime` is the best way to call it. 
I realise it matches the internal datapoint, but that doesnt have to be explicit to consumers.

However I dont have a better idea, what do you think?

Reply:
I am also a bit unsure
as I think this function is mostly for the speaker timer the convention from other apps like stagetimer.io seams to be add/remove time